sdk/lib/core/uri.dart

```diff
@@ -50,14 +78,22 @@ const String _hexDigits = "0123456789ABCDEF";
 /// ```
 ///
 /// ## HTTP and HTTPS URI
-/// To create a URI with https scheme, use [Uri.https] or [Uri.http]:
+/// As mentioned before, these are special constructors that do the same thing
+/// as the standard constructor, but with less typing involved.
+/// To create a URI with https scheme, use [Uri.https]:
 /// ```dart
 /// final httpsUri = Uri.https('example.com', 'api/fetch', {'limit': '10'});
 /// print(httpsUri); // https://example.com/api/fetch?limit=10
 /// ```
+/// 
+/// For the old http scheme, use [Uri.http]:
+/// 
 /// ## File URI
-/// To create a URI from file path, use [Uri.file]:
+/// To create a URI for a file path, use [Uri.file]:
 /// ```dart
+/// // for your convenience, use r before the string to deactivate
```

Comment:
Not the best place to explain raw strings in detail. It's OK to mention them briefly, and link to `[String]` instead for details.
Maybe:
```dart
/// (Uses raw strings ([String]) for paths to avoid the `\`s of Windows paths being
/// interpreted as a string escape sequence.)
```